gpiozero/pins/mock.py

```diff
@@ -413,3 +385,34 @@ def tx_word(self, value, bits_per_word=None):
             bits = reversed(bits)
         self.tx_buf.extend(bits)
 
+
+class MockFactory(LocalPiFactory):
+    def __init__(self, revision='a21041', pin_class=MockPin):
+        super(MockFactory, self).__init__()
+        self._revision = revision
+        self.pin_class = pin_class
+
+    def _get_address(self):
+        return ('mock',)
+
+    def _get_revision(self):
+        return self._revision
+
+    def reset(self):
```

Comment:
Well, I figured it's called externally (by `test_mock_pin.teardown_function`) so it's effectively part of the "public" API. It may not be part of the parent's public API (or any other descendant's) but then it's a fairly special descendant so I don't have any particular issue with it having a slightly "enhanced" API compared to the rest.